### PR TITLE
Added analyzer feedback files for pylint generic comments & edited other comments to conform to spec.

### PR DESCRIPTION
**Pull request by BethanyG**
Added `website-copy/analyzer-comments/python` As part of work on the first-pass refactor of the [Python Analyzer](https://github.com/exercism/python-analyzer/pull/22).

- [x] Added `python/pylint` directory + `pylint` comment markdown files.
- [x] Added `python/general` directory + edited the general comment markdown files.
- [x] Added `python/two-fer` directory + edited the `two-fer` comment markdown files.
- [x] Added title headers to all markdown files.

### CODE REVIEW

**Review by SleeplessByte — APPROVED** [review]

File: analyzer-comments/python/general/no_return.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# general missing return statement
+
+A `return` statment is not being used in this function to return the result string, and this code should have failed one or more test cases during testing.
```
Comment (SleeplessByte) [inline analyzer-comments/python/general/no_return.md:3]: Is it always a "result string"?

File: analyzer-comments/python/pylint/convention.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# pylint convention
+
+PyLint is reporting that this code doesn't follow general code style conventions.
+While this type of issue generally doesn't affect the way your code runs, it can hurt
+readability or automated tools such as automatic documentors or test runners.
+
+On line #%{lineno}, %{code} was reported:  %{message}.
```
Comment (SleeplessByte) [inline analyzer-comments/python/pylint/convention.md:7]:
```suggestion
On line #%{lineno}, %{code} was reported: %{message}.
```

File: analyzer-comments/python/pylint/convention.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+For more information on PyLint and PyLint checker message code %{code} see:
+*  [PyLint Format Checker](http://pylint.pycqa.org/en/latest/technical_reference/features.html#format-checker-messages)
```
Comment (SleeplessByte) [inline analyzer-comments/python/pylint/convention.md:10]:
```suggestion

*  [PyLint Format Checker](http://pylint.pycqa.org/en/latest/technical_reference/features.html#format-checker-messages)
```

File: analyzer-comments/python/pylint/error.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
+# pylint informational
+
+PyLint is reporting that it has found an error or problem in the code that needs to be addressed.
+
+On line #%{lineno}, %{code} was reported:  %{message}.
```
Comment (SleeplessByte) [inline analyzer-comments/python/pylint/error.md:5]:
```suggestion
On line #%{lineno}, %{code} was reported: %{message}.
```

Are you sure you want to `.` in there at the end?

File: analyzer-comments/python/pylint/convention.md (hunk 2 above)
Comment (SleeplessByte) [reply analyzer-comments/python/pylint/convention.md:7]: Are you sure you want the dot in there at the end?

File: analyzer-comments/python/pylint/refactor.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
+# pylint refactor
+
+PyLint is reporting that this code has a [code smell](https://en.wikipedia.org/wiki/Code_smell), and might be in need of
+a re-write or refactor. This doesn't mean your code is _incorrect_ or _buggy_ in a technical sense -- only that it
+appears to have one or more patterns that could lead to _future_ bugs or maintenance issues, and you should take a closer look at it.
+
+On line #%{lineno}, %{code} was reported: %{message}.
```
Comment (SleeplessByte) [inline analyzer-comments/python/pylint/refactor.md:7]: Are you sure you want the `.` in there?

File: analyzer-comments/python/pylint/warning.md [hunk 6]
```diff
@@ -0,0 +1,11 @@
+# pylint warning
+
+PyLint is reporting that it found an issue in the code that could lead to a bug or error in the program.  
+While this error might not be severe, it could lead to more severe issues in the future.  
+We recommend addressing the problem before proceeding further.
+
+On line #%{lineno}, %{code} was reported: %{message}.
```
Comment (SleeplessByte) [inline analyzer-comments/python/pylint/warning.md:7]: Guess what I'm going to say :P

File: analyzer-comments/python/two-fer/conditionals.md [hunk 7]
```diff
@@ -0,0 +1,7 @@
+# unnecessary conditionals
+
+[Conditionals](https://github.com/exercism/python/blob/main/concepts/conditionals/introduction.md) are unnecessarily used in this solution.
+It can be improved by making use of a `default argument` in combination with either [`str.format`](https://docs.python.org/3/library/stdtypes.html#str.format) or [`f-strings`](https://docs.python.org/3/reference/lexical_analysis.html#formatted-string-literals) (_otherwise known as `formatted string literals` or `interpolated strings`_).
+
+See this post on [python default arguments](https://blog.finxter.com/python-default-arguments/) for more details.
```
Comment (SleeplessByte) [inline analyzer-comments/python/two-fer/conditionals.md:6]: The extra spaces at the end would probably trigger format warnings when we add them.

```suggestion
See this post on [python default arguments](https://blog.finxter.com/python-default-arguments/) for more details.
```

File: analyzer-comments/python/two-fer/no_method.md [hunk 8]
```diff
@@ -0,0 +1,5 @@
+# no method or function definition found
```
Comment (SleeplessByte) [inline analyzer-comments/python/two-fer/no_method.md:1]: I recommend making this a general comment and give `two_fer` as argument.

File: analyzer-comments/python/two-fer/no_module.md [hunk 9]
```diff
@@ -0,0 +1,4 @@
+# missing module or misnamed file
+
+We were unable to find a file called `two_fer.py` in your file path.
```
Comment (SleeplessByte) [inline analyzer-comments/python/two-fer/no_module.md:3]: I recommend making this a general comment and give `two_fer.py` as argument.

File: analyzer-comments/python/two-fer/percent_formatting.md [hunk 10]
```diff
@@ -0,0 +1,8 @@
+# percent or "old style" formatting
```
Comment (SleeplessByte) [inline analyzer-comments/python/two-fer/percent_formatting.md:1]: You might want to make this a general comment for any instance in any exercise.

File: analyzer-comments/python/general/no_return.md (hunk 1 above)
Comment (BethanyG) [reply analyzer-comments/python/general/no_return.md:3]: ah!  good catch.  No.  Not for general feedback.  I was in my `two-fer` mode.  LOL.  Will note for future cleanup.  😄 .

**Comment by BethanyG** [comment]
@SleeplessByte  -- thank you so much for these suggestions!  I will submit a follow-on PR to address them.  😄

**Pull request merged**